app/controllers/class_categories_controller.rb

```diff
@@ -0,0 +1,66 @@
+class ClassCategoriesController < ApplicationController
+  before_action :find_class_category, only: %i[edit update show destroy]
+  before_action :authenticate_user!
+  def index
+    @class_categories = ClassCategory.all
+  end
+
+  def edit
+    all_teachers
+  end
+
+  def update
+    if @class_category.update(class_category_params)
+      flash[:notice] = 'Categoria de Aula editada com sucesso.'
+      redirect_to class_category_path(class_category_params)
+    else
+      all_teachers
+      flash[:notice] = 'Ocorreram erros durante a edição, veja abaixo:'
```

Comment:
```suggestion
      flash.now[:notice] = 'Ocorreram erros durante a edição, veja abaixo:'
```

Acho que aqui como é render talvez o ideal fosse ser flash.now o que acha?